users/models.py

```diff
@@ -111,3 +143,24 @@ class LegalAddress(TimestampedModel):
     def __str__(self):
         """Str representation for the legal address"""
         return f"Legal address for {self.user}"
+
+
+class Profile(TimestampedModel):
+    """ A user's profile and employment info """
+
+    user = models.OneToOneField(User, on_delete=models.CASCADE, related_name="profile")
+
+    gender = models.CharField(max_length=10, choices=GENDER_CHOICES)
+    birth_year = models.IntegerField()
+
+    company = models.CharField(max_length=128)
+    company_size = models.IntegerField(
+        null=True, blank=True, choices=COMPANY_SIZE_CHOICES
+    )
+    industry = models.CharField(max_length=60, blank=True)
+    job_title = models.CharField(max_length=128)
```

Comment:
Nit: can you define this after `company`? Would be nice to keep the required fields together, and it matches the order of entry in the form